### ISSUE TITLE
!getgames clears lobby subscriber list

### ISSUE THREAD

**Issue by jmrico01** [impossible-bosses/ibce-bots#31]
Running `!getgames` after people have subscribed to a lobby will clear the subscriber list.

**Comment by Archimonde666** [comment]
I noticed this and actually think it was a feature : you don't want to repeat the mention to the players each time the lobby is refreshed
let it be

**Comment by jmrico01** [comment]
I don't think you understand. It doesn't mention when the lobby is refreshed, it mentions when the lobby starts. So if you subscribe to the lobby wanting to be notified when it starts, then I run `!getgames`, it will remove you and you won't get the notification you wanted.

**Comment by Archimonde666** [comment]
ah ye i wasn't understanding that, i though it was about the mentions of the :okib: players that gets called after 8 (it's me) and also after game is hosted (that what i though it was)
in that case sure it need fix ^^'
